CHANGELOG.md

```diff
@@ -16,19 +16,97 @@ Also, any bug fix must start with the prefix �Bug fix:� followed by the desc
+
+ - SKEXP0001: 
+   - `Encamina.Enmarcha.SemanticKernel`   
+ - SKEXP0003: 
+   - `Encamina.Enmarcha.SemanticKernel`
+   - `Encamina.Enmarcha.SemanticKernel.Abstractions`
+   - `Encamina.Enmarcha.SemanticKernel.Connectors.Memory`
+   - `Encamina.Enmarcha.SemanticKernel.Plugins.Memory`
+   - `Encamina.Enmarcha.SemanticKernel.Plugins.QuestionAnswering`
+ - SKEXP0011:
+   - `Encamina.Enmarcha.SemanticKernel.Connectors.Memory`
+ - SKEXP0026: 
+   - `Encamina.Enmarcha.SemanticKernel.Connectors.Memory`
+ - SKEXP0051:
+   - `Encamina.Enmarcha.SemanticKernel.Connectors.Document`
+
+ More information about these warnings is available here: https://github.com/microsoft/semantic-kernel/blob/main/dotnet/docs/EXPERIMENTS.md
+
+### Braking Changes
+
+- Replaced class `Completition` for `Completion` in `Encamina.Enmarcha.AI.OpenAI.Abstractions`. It was misspelled.
+- Class `SemanticKernelOptions` does not exists anymore. It has been replaced by `AzureOpenAIOptions` from `Encamina.Enmarcha.AI.OpenAI.Abstractions`. 
+- The following references were updated due to changes in `Semantic Kernel` version `1.0.1`:
+ - Changed `IKernel` for `Kernel`.
+ - Changed `ISKFunction` for `KernelFunction` or `KernelPlugin`.
+ - Changed `SKFunction` for `KernelFunction`.
+ - Changed `ContextVariables` for `KernelArguments`.
+ - Changed `kernel.Functions.GetFunction(...)` for `kernel.Plugins[<name of plugin>][<name of function>]`.
+ - Changed `OpenAIRequestSettings` for `OpenAIPromptExecutionSettings`.
+- Removed extension methods for `SKContext` because that class does not exists anymore in `Semantic Kernel`.
+- Due to the breaking nature of the new version of `Semantic Kernel`, the following extension methods are not available any more and have been replace by new methods, and it was not possible to marked it as `Obsolete`:
+  - `GetSemanticFunctionPromptAsync` is replaced by `GetKernelFunctionPromptAsync`.
+  - `GetSemanticFunctionUsedTokensAsync` is replaced by `GetKernelFunctionUsedTokensAsync`.
+  - `ImportSemanticPluginsFromAssembly` is replaced by `ImportPromptFunctionsFromAssembly`.
+- Extension method `GetSemanticFunctionPromptAsync` is no longer available. It is replaced by `GetKernelFunctionPromptAsync`. 
+- Extension method `ImportQuestionAnsweringPlugin` has different signature.
+- Extension method `ImportQuestionAnsweringPluginWithMemory` has different signature.
+- Extension method `ImportChatWithHistoryPluginUsingCosmosDb` has different signature.
+- The format of prompt function configuration files `config.json` has been modified.
+
+### Major Changes
+
+- Updated `Semantic Kernel` from `1.0.0-beta8` to `1.0.1` (first final version of `Semantic Kernel`).
```

Comment:
Semantic kernel is updated to version 1.1.0, right?

Reply:
Yep. That is a typo.

Reply:
It should be fixed by now.